### ISSUE TITLE
Not working with multiple cursors

### ISSUE THREAD

**Issue by Noitidart** [albertromkes/tabout#7]
I have multiple cursors, all of them lined up on a quote. Hitting tab removes all other cursors and just tabs through the first cursors quote.

Possible solutions:
* If all multi cursors are lined up, then tab through. Else don't.

**Comment by gaetawoo** [comment]
@Noitidart @albertromkes 

To everyone here, adding the following to your _keybindings.json_ file will fix this behavior. You'll be able to have Tabout enabled **AND** cycle through snippet tab stops **AND** use tab with multiple cursors lined up on a quote or other tabout-able character:
```
[
    { "key": "tab","command": "-tabout",
    "when": "editorTextFocus && !suggestWidgetVisible"},
    { "key": "tab","command": "tabout",
    "when": "editorTextFocus && !suggestWidgetVisible && !inSnippetMode && !editorHasMultipleSelections"} 
    ]```

**Comment by albertromkes** [comment]
Fixed in 0.1.5 Thanks @gaetawoo

**Comment by Noitidart** [comment]
Thank you!!!

**Comment by gaetawoo** [comment]
@albertromkes I'm actually getting an error now that `command '-tabout' not found`

This didn't happen in the previous version of Tabout when I used my keybindings.json file to fix the problems. But now, that doesn't work when it's in the official keybindings settings file. 

I should have been more clear that I was not able to test the changes in the pull request, I have no idea how to build the extension. But the keybindings.json entries worked and I figured they could be replaced in the package.json.
